data/campaigns/Winds_of_Fate/scenarios/12_Epilogue.cfg

```diff
@@ -9,23 +9,14 @@
 
     [story]
         [part]
-            story= _ "Dominant Karron heard the Memories Spoken to her by the gliders. She lead several flights west to the Green Isle of the orcs. Striving together with the noble creatures of this far land, the infestation of orcs was destroyed so new eyries could be set there."
+            story= _ "Banished from the Greatland, Dominant Karron led several flights west. The next few flights that arrived on the Greatland reported that some drakes had joined her, planning to build new eyries overlooking the limitless prey of the Green Isle."
         [/part]
         [part]
             story= _ "Countless drakes flocked from Morogor to scorch the Greatland with their need. Those creatures that survived their hunt formed large hosts with a deadly cunning. Soon, whole flights fell to their own prey. Others were scattered far across the Greatland as more poured over from Morogor to supplant them."
         [/part]
         [part]
             story= _ "As most grew weary of the hunt, Dominant Gorlack summoned all flights to a counsel gathering high upon his Sunreach Peaks. There, he shared with them a new Windpath, the <b>Great Contention</b>. By its wisdom, any creatures that spoke with an honest tongue, that respected the domain of another, that obeyed our Ways... might not be taken as prey. Such creatures would be justly feared as fellow contenders for world dominance."
         [/part]
-        [part]
-            story= _ "In accord, the flights went again into exile, withdrawing to the Greatland’s many remote places. So ended the <i>War with Prey</i>."
```

Comment:
Can we keep this line? It's a nice closer.

Reply:
Sure, it's only the first line that I'm really bothered about.